components/carousel/index.tsx

```diff
@@ -106,7 +108,11 @@ export default class Carousel extends React.Component<CarouselProps, {}> {
     const dotsClass = 'slick-dots';
     const dotPosition = this.getDotPosition();
     props.vertical = dotPosition === 'left' || dotPosition === 'right';
-    props.dotsClass = `${dotsClass} ${dotsClass}-${dotPosition || 'bottom'}`;
+    props.dotsClass = classNames(
+      dotsClass,
+      this.props.dotsClass,
+      `${dotsClass}-${dotPosition || 'bottom'}`,
+    );
```

Comment:
@afc163 `dotsClass` Should it be replaced or supplemented？🤔

Reply:
it's same as the previous one.

Reply:
Your changes is to supplement it, not replace it.

Maybe it should be like this:
```js
props.dotsClass = classNames(
    this.props.dotsClass,
    `${this.props.dotsClass}-${dotPosition || 'bottom'}`,
);
```
And then provide all the styles yourself need. 

My personal opinion.

Reply:
The reason behind doing this is we don't want to lose the whole style provided by slick slider. By simply applying an external class name we can overwrite that style easily.

Reply:
Ok, Let the owner to decide it.

Reply:
ok

Reply:
I prefer patch additional className instead of modify origin provided

Reply:
Do I make any changes in this pull request or not?

Reply:
any update about this PR?

Reply:
https://github.com/ant-design/ant-design/pull/21156#issuecomment-581262408